test/Microsoft.AspNet.Mvc.Core.Test/Rendering/DefaultTemplatesUtilities.cs

```diff
@@ -191,11 +192,17 @@ public static HtmlHelper<TModel> GetHtmlHelper<TModel>(TModel model, ICompositeV
                     GetAntiForgeryInstance(),
                     bindingContextAccessor,
                     provider,
-                    urlHelper);
+                    urlHelper,
+                    new HtmlEncoder());
             }
 
             // TemplateRenderer will Contextualize this transient service.
-            var innerHelper = (IHtmlHelper)new HtmlHelper(htmlGenerator, viewEngine, provider);
+            var innerHelper = (IHtmlHelper)new HtmlHelper(htmlGenerator,
+                                                          viewEngine,
```

Comment:
nit: dont add all the extra spaces, just have all parameters lie 1 indent in as if you hit enter on the previous line (much easier to maintain in the future). Do the same for other locations in this PR.